docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -39,6 +39,30 @@ The following illustration describes the workflow for live migration using YB Vo
 
 ![fall-back-workflow](/images/migrate/fall-back-workflow-new.png)
 
+| Step | Description |
```

Comment:
Tables are fine. More general comment for the future - I find these tables too long, and they don't quite match up with either the flow diagram above, or the page headings/structure below. I think they should explicitly match the diagram - including the way it is broken into stages, and the page should also match this structure.